goimports: add page (#3308)
tldr-pages/tldr: pages/common/goimports.md

# goimports

> > Updates your Go import lines, adding missing ones and removing unreferenced ones.

Reviewer: ```suggestion
> Updates Go import lines, adding missing ones and removing unreferenced ones.
```
Let's keep it impersonal where possible.

> > More information: https://godoc.org/golang.org/x/tools/cmd/goimports.

Reviewer 2: Small formatting correction:
```suggestion
> More information: <https://godoc.org/golang.org/x/tools/cmd/goimports>.
```

> - Display your completed import source file:

Reviewer 2: I think we want to avoid using `your` here. What about this?
```suggestion
- Display the completed import source file:
```

> - Display diffs instead of rewriting file:

Reviewer: ```suggestion
- Display diffs instead of rewriting the original file:
```

> - Display diffs instead of rewriting file:

Reviewer 3: We should have a basic goimports file.go invocation as the first example.

`goimports -d {{file}}.go`

> - Write result to (source) file instead of stdout:

Reviewer 2: Perhaps this will read better?
```suggestion
- Write the result to the source file instead of the standard output:
```

> - Write the result to the source file instead of the standard output:

Reviewer: ```suggestion
- Write the result back to the source file instead of the standard output:
```

`goimports -w {{file}}.go`

> - Display diffs and write result to (source) file:

Reviewer 2: Perhaps this will read better?
```suggestion
- Display diffs and write the result to the source file:
```

> - Display diffs and write the result to the source file:

Reviewer: ```suggestion
- Display diffs and write the result back to the source file:
```
